src/admin/shared/helpers/content-picker/parse-picker.ts

```diff
@@ -1,6 +1,32 @@
 import { ContentPickerType, PickerItem } from '../../types';
 
+const parseSearchQuery = (input: string) => {
+	// replace %22 by "
+	const replacedString = input.split('%22').join('"');
```

Comment:
Kan je hier niet decodeURI() gebruiken om die %22 te resolven?

Aangezien die decodeURI geen effect heeft op een uri die reeds decoded is